src/message/__tests__/messageActionSheet-test.js

```diff
@@ -56,60 +56,98 @@ describe('constructActionButtons', () => {
   });
 });
 
-describe('constructHeaderActionButtons', () => {
+describe('constructTopicActionButtons', () => {
+  test('show mark as read if topic is unread', () => {
+    const stream = eg.makeStream({ name: 'test stream', stream_id: 123 });
```

Comment:
Similarly, instead of explicitly setting `name: 'test stream'` and then writing `test stream` again later in the test (in one of the intermediate commits in this branch), we can just say `stream.name` at that later point.

That makes the test code simpler and shorter -- and in particular it makes it clear that there's nothing interesting about the particular name and it isn't something the reader needs to pay attention to.

That at that point we're passing no arguments to `eg.makeStream` at all, so we might simplify further by using `eg.stream` instead of a local, to further make clear that there's nothing interesting about this particular stream.